Week2/homework/round.js

```diff
@@ -20,17 +20,11 @@ console.log(a);
 
 
 //5.5 So now we have z and a find a way to compare the two values and store the highest of the two in a new variable.
- let h;
- if (z > a)
+let max = Math.max(a, z);
```

Comment:
Nice! I would suggest using `const` though (because the value of `max` will not change anymore

Reply:
yes definitely!